db/generate_data.js

```diff
@@ -18,35 +18,51 @@ let count = 0;
 let rating = 0;
 let fitFeedback = 0;
 
-gender.forEach(gender => {
-  material.forEach(material => {
-    action.forEach(action => {
-      reviews.forEach((review, index) => {
-        let tempRating = randomNumberGenerator(1, 5);
-        let tempFitFeedback = randomNumberGenerator(-1, 1);
-        rating += tempRating;
-        fitFeedback += tempFitFeedback;
-        count += 1;
-        reviewBulkData.push({ name: faker.name.firstName(), headline: headlines[index], review: review, rating: tempRating, fit_feedback: tempFitFeedback, shoe_id: model });
+async function generateAndInsertBatches() {
+  for (var i = 0; i < 100000; i++) {
```

Comment:
It looks like the insert batches function is only run once in this file, so if you're running it from a script or another file how are you making the model number not start over at 1 when you run the next batch?